app/views/vet/confirmation.njk

```diff
@@ -11,13 +11,13 @@
         html: "Booking reference number<br><strong>" + reference + "</strong>"
       }) }}
 
-      <p>The record has been submitted.</p>
       <h2 class="govuk-heading-m">What happens next</h2>
-      <p class="govuk-body">We’ve contacted the farmer to ask them to check and confirm the record. They can claim their funding when they have confirmed this.</p>
+      <p class="govuk-body">We’ve contacted the farmer to tell them they can claim their funding.</p>
       <p class="govuk-body">Invoice the farmer if you have not already done this.</p>
       <h2 class="govuk-heading-m">Contact the Rural Payments Agency</h2>
       <p class="govuk-body">If you have a question about your submission, call the Rural Payments Agency on 03000 200 301</p>
       <p class="govuk-body">Monday to Friday 8.30am to 5pm, except bank holidays <a href="https://www.gov.uk/call-charges" target="_blank">Find out about call charges</a>.</p>
+      <p class="govuk-body"><a href="https://www.gov.uk/call-charges" target="_blank">What did you think of this service?</a>(takes 30 seconds)</p>
```

Comment:
I don't think the href should be `../call-charges`, please update to whatever it should be. There needs to be a space before the opening paren.

Reply:
Thats not been given yet. I have asked Joy this question in ticket as well.